lib/elastic_record/index.rb

```diff
@@ -65,6 +66,16 @@ def enable!
       @disabled = false
     end
 
+    def load_from_source(&block)
+      if block_given?
+        @load_from_source = true
+        yield
+        @load_from_source = false
+      else
+        @load_from_source
+      end
+    end
```

Comment:
I'd rather think about the block style API a bit more. Do you need this yet?

Reply:
I'm only using it for the tests, but it isn't necessary